cmd/thanos/query.go

```diff
@@ -403,3 +423,21 @@ func removeDuplicateStoreSpecs(logger log.Logger, duplicatedStores prometheus.Co
 	}
 	return deduplicated
 }
+
+type stringSlice []string
+
+// duplicate returns a duplicate string in the given stringSlice
+// or empty string if none was found.
+func (ss stringSlice) duplicate() string {
```

Comment:
What about just passing slice? Not sure if it's worth to create custom stringSlice type for it, cannot see advantage

Reply:
it was just easier for my eyes, I have no strong opinions here, will change to parameter instead of type :+1:

Reply:
fixed